src/job/job.cpp

```diff
@@ -238,6 +238,13 @@ void job::process_task_limits(std::shared_ptr<sandbox_limits> limits)
 			throw job_exception("memory" + msg);
 		}
 	}
+	if (limits->extra_memory == SIZE_MAX) {
+		limits->extra_memory = worker_limits.extra_memory;
+	} else {
+		if (limits->extra_memory > worker_limits.extra_memory) {
+			throw job_exception("extra-memory" + msg);
```

Comment:
How about a space after "memory"?

Reply:
![No Meme](https://imgflip.com/s/meme/How-About-No-Bear.jpg)

Reply:
Yeah, i didn't think of checking 40 more lines to the top. Sorry